packages/storybook/stories/Stateful Client/Adapters.stories.mdx

```diff
@@ -0,0 +1,136 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import { Canvas, Description, Heading, Props, Source, Title } from '@storybook/addon-docs/blocks';
+
+<Meta id="statefulclient-reacthooks-setup" title="Stateful Client/React Hooks/Adapters" />
+
+# Adapters for Composites
+
+Composites deliver turn-key communication experiences for developers.
+They make it easy for developers to add end-to-end communication experiences to their apps with only a couple lines of code.
+To power Composites, the `communication-react` library provides `Adapters`, one for calling and one for chat Composites.
+These adapters make use of the stateful clients for calling and chat in conjunction with React Hooks like usePropsFor, to provide developers a single interface to configure Composites.
```

Comment:
Remove `usePropsFor` here as this hook isn't used to configure composites (just used to integrate components directly with stateful client).
Maybe could mention something like `in conjunction with other exposed properties to provide...` but our others properties are a bit of mismash of configurable things